src/apps/subjects/tests/tests.py

```diff
@@ -247,6 +247,7 @@ class TestSubjects(BaseTest):
 
     login_url = "/auth/login"
     subject_list_url = "/subjects"
+    my_subject_url = "/users/me/subjects/{applet_id}"
     subject_detail_url = "/subjects/{subject_id}"
```

Comment:
to follow the restfull pattern, this endpoint should be `/subjects/applet/{applet_id}/me`

or `/applets/{applet_id}/subject`

Reply:
I understand. I was following the established pattern for the users router

<img width="480" alt="image" src="https://github.com/user-attachments/assets/9d9820d7-b4df-4bff-8e45-eee266498906">

Do you think I should change it?

Reply:
Ok, let keep the pattern we already have, no need to change

Reply:
But you should implement it in users resource, not in subject, call subject service from users resource

Reply:
Fair enough. I was following the example from the schedule router [here](https://github.com/ChildMindInstitute/mindlogger-backend-refactor/blob/4bef04b904374e539a59bead67ab93aaed52b3e8/src/apps/schedule/router.py#L182), but I think I'll move it to the user router